helm.yml

```diff
@@ -28,16 +28,17 @@ grafana.ini:
     oauth_auto_login: true
     disable_login_form: false
     signout_redirect_url: https://{{IngressHost}}/Account/Logout
-  auth.generic_oauth:
+  auth.azuread:
+    name: Azure AD
     enabled: true
-    client_id: {{Auth0ClientId}}
-    client_secret: {{Auth0ClientSecret}}
-    scopes: openid profile email
-    auth_url: https://{{Auth0Domain}}/authorize
-    token_url: https://{{Auth0Domain}}/oauth/token
-    api_url:  https://{{Auth0Domain}}/usrinfo
-    send_client_credentials_via_post: true
     allow_sign_up: true
+    client_id: {{AdOidcClientId}}
+    client_secret: {{AdOidcClientSecret}}
+    scopes: openid email profile
+    auth_url: https://login.microsoftonline.com/{{AzureCDSTenantId}}/oauth2/v2.0/authorize
+    token_url: https://login.microsoftonline.com/{{AzureCDSTenantId}}/oauth2/v2.0/token
```

Comment:
Would FCS be more sensible here if we're to add anything Production related in future?